src/main/resources/org/damap/base/template/horizonEuropeTemplate.resource

```diff
@@ -20,6 +20,10 @@ restrictedAccess.avail=Description of protocol to access restricted data:
 costs.avail=There are costs dedicated to data management and ensuring that data will be FAIR as outlined below.
 costs.no=There are no costs dedicated to data management and ensuring that data will be FAIR.
 
+storageIntro.internal=For the duration of the project, storage and backup of data will be ensured by [name] (acting as the person responsible for data management and DMP) in cooperation with the system operator. The data will be stored on the servers of our institution.
+storageIntro.external=For the duration of the project, storage and backup of data will be ensured by [name] (acting as the person responsible for data management and DMP) in cooperation with the system operator. The data will not be stored on the servers of our institution.
+storageIntro.both=For the duration of the project, storage and backup of data will be ensured by [name] (acting as the person responsible for data management and DMP) in cooperation with the system operator. The data will be stored both on the servers of our institution and external storage.
```

Comment:
Would something like a note to look at the details in the storage table make sense? e.g. `Details about storage options and the stored datasets can be found in the storage table.`

Right now, the wording might suggest that everything is stored on internal __as well as__ external storage. Suggestion: `External and internal storage options will be used. <sentence from above about storage details>`. Just a first draft though

Reply:
Done